src/app/components/book/book.component.html

```diff
@@ -1,77 +1,98 @@
 
-    <p
-      class="font-tilt-neon h-[5%] text-pale-brown text-lg font-semibold flex self-start"
-    >
-      Synopsis
-    </p>
-    <p class="w-full h-1/5 font-sans font-light text-pale-brown text-md">
-      This is the description of a very good book that should definitely be
-      read. This book has a captivating plot with several unexpected and
-      unpredictable twists and great character development.
-    </p>
-    <p
-      class="font-tilt-neon h-[5%] text-pale-brown text-lg font-semibold flex self-start"
-    >
-      Mood
-    </p>
-    <p
-      class="bg-amaranth text-white rounded-3xl w-fit px-2 py-0.5 flex self-start"
-    >
-      Joy
-    </p>
-    <p
-      class="font-tilt-neon h-[5%] text-pale-brown text-lg font-semibold flex self-start"
-    >
-      Pace
-    </p>
-    <p
-      class="bg-amaranth text-white rounded-3xl w-fit px-2 py-0.5 flex self-start"
-    >
-      Fast
-    </p>
-    <p
-      class="font-tilt-neon h-[5%] text-pale-brown text-lg font-semibold flex self-start"
-    >
-      Status
-    </p>
-    <p
-      class="bg-amaranth text-white rounded-3xl w-fit px-2 py-0.5 flex self-start"
-    >
-      Completed
-    </p>
-    <p
-      class="font-tilt-neon h-[5%] text-pale-brown text-lg font-semibold flex self-start"
-    >
-      Comments
-    </p>
-    <div
-      class="comments-container py-2 h-max w-full flex flex-col gap-2 justify-start items-center align-start"
-    >
-      @for (comment of mockComments; track comment){
-      <p class="bg-linen w-full h-fit px-4 py-2 rounded-lg">{{ comment }}</p>
+          </div>
+          <p
+            class="font-tilt-neon h-fit w-fit text-pale-brown text-lg text-center font-semibold"
+          >
+            {{ book.title }}
+          </p>
+          <p class="font-tilt-neon h-fit w-fit text-amaranth text-sm">
+            {{ book.author }}
+          </p>
+          <p class="font-tilt-neon h-fit w-fit text-pale-brown text-md">
+            {{ book.pages }} pages
+          </p>
+        </div>
+
+        <p
+          class="font-tilt-neon h-[5%] text-pale-brown text-lg font-semibold flex self-start"
+        >
+          Synopsis
+        </p>
+        <p class="w-full h-1/5 font-sans font-light text-pale-brown text-md overflow-y-scroll mb-4">
+          {{
+            book.description
+          }}
+        </p>
+        @if (book.moods[0] != "") {
+          <p
+            class="font-tilt-neon h-[5%] text-pale-brown text-lg font-semibold flex self-start"
+          >
+            Moods
+          </p>
+
+          @for (mood of book.moods; track mood) {
+            <p
+              class="bg-amaranth text-white rounded-3xl w-fit px-2 py-0.5 flex self-start"
+            >
+              {{ mood }}
+            </p>
+          }
+        }
+
+        @if (book.pace != "") {
+          <p
+            class="font-tilt-neon h-[5%] text-pale-brown text-lg font-semibold flex self-start"
```

Comment:
I intend to try and refactor this when I am finalising the project, currently I have to do a lot of safety checks to prevent errors and handle the way my state and database is set up.